modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -501,16 +502,22 @@ def get_result(self, handle: ResultHandle, **kwargs: KwargTypes) -> BackendResul
             self._update_cache_result(handle, backres)
             return backres
 
-    def cost_estimate(self, circuit: Circuit, n_shots: int) -> float:
-        """
-        Estimate the cost in HQC to complete this `circuit` with `n_shots` repeats. The
-        estimate is based on hard-coded constants, which may be out of date,
-        invalidating the estimate. Use with caution.
+    def cost_estimate(self, circuit: Circuit, n_shots: int) -> Optional[float]:
+        """Deprecated, use ``cost``."""
+
+        warnings.warn(
+            "cost_estimate is deprecated, use cost instead", DeprecationWarning
+        )
 
-        With 𝑁1𝑞 PhasedX gates, 𝑁2𝑞 ZZMax gates, 𝑁𝑚 state preparations and measurements,
-        and 𝐶 shots:
+        return self.cost(circuit, n_shots)
 
-        𝐻𝑄𝐶= 5 + (𝑁1𝑞 + 10𝑁2𝑞 + 5𝑁𝑚)*𝐶/5000
+    def cost(self, circuit: Circuit, n_shots: int) -> Optional[float]:
+        """
+        Return the cost in HQC to complete this `circuit` with `n_shots`
```

Comment:
What does HQC means here?

Reply:
I think now it stands for "H-series quantum credits" it's the terminology used on the user portal